src/web/topic/components/Indicator/DeleteIndicator.tsx

```diff
@@ -0,0 +1,26 @@
+import { Delete } from "@mui/icons-material";
+
+import { deleteEdge, deleteNode } from "../../store/createDeleteActions";
+import { Indicator } from "./Indicator";
+
+interface Props {
+  graphPartId: string;
+  graphPartType: string;
+}
+
+export const DeleteIndicator = ({ graphPartId, graphPartType }: Props) => {
+  return (
+    <Indicator
+      Icon={Delete}
+      title={"Delete"}
+      onClick={ () => {
```

Comment:
prettier is angry here. there's a precommit hook to fix whitespace errors, but you can also use `npx prettier --write .` if you prefer